src/worker.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var queue = require('queue-async');
+var map = require(process.argv[2]);
+
+var q = queue();
+var sources = [];
+
+JSON.parse(process.argv[3]).forEach(function(source) {
+  q.defer(loadSource, source);
+});
+
+function loadSource(source, done) {
+  if (source.mbtiles) sources.push({name: source.name, getTile: require('./mbtiles')(source, done)});
```

Comment:
I wonder if there is a way to make this an interface where any source can be used the same way, without knowing the implementation details (this could help us with additional plugins down the road). Perhaps instead of returning the getTile function synchronously, we should send it back as the result of the `ready` callback. I believe that would allow users of the parse interfaces to treat them the same.